packages/react-router-dom/docs/guides/quick-start.md

```diff
@@ -181,4 +181,4 @@ function Topic() {
 
 ## Keep Going!
 
-Hopefully these examples give you a feel for what it's like to create a web app with React Router. Keep reading to learn more about [the primary components](primary-components.md) in React Router!
+Hopefully these examples give you a feel for what it's like to create a web app with React Router. Keep reading to learn more about [the primary components](web/guides/primary-components) in React Router!
```

Comment:
This breaks the docs on GitHub. I don't recall how  we map links when we build the website, but would changing `primary-components.md` to `./primary-components.md` work?

Reply:
Hey, thanks for the feedback!

I'm willing to check and adapt it myself, but I'm having these errors when I attempt to start a local website following the README.

```
ERROR in ./modules/index.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
ReferenceError: Unknown plugin "@babel/proposal-class-properties" specified in "/Users/williamghelfi/Work/React/react-router/website/modules/.babelrc" at 1, attempted to resolve relative to "/Users/williamghelfi/Work/React/react-router/website/modules"
    at /Users/williamghelfi/Work/React/react-router/website/node_modules/babel-core/lib/transformation/file/options/option-manager.js:180:17
    at Array.map (<anonymous>)
    at Function.normalisePlugins (/Users/williamghelfi/Work/React/react-router/website/node_modules/babel-core/lib/transformation/file/options/option-manager.js:158:20)
    at OptionManager.mergeOptions (/Users/williamghelfi/Work/React/react-router/website/node_modules/babel-core/lib/transformation/file/options/option-manager.js:234:36)
    at OptionManager.init (/Users/williamghelfi/Work/React/react-router/website/node_modules/babel-core/lib/transformation/file/options/option-manager.js:368:12)
    at File.initOptions (/Users/williamghelfi/Work/React/react-router/website/node_modules/babel-core/lib/transformation/file/index.js:212:65)
    at new File (/Users/williamghelfi/Work/React/react-router/website/node_modules/babel-core/lib/transformation/file/index.js:135:24)
    at Pipeline.transform (/Users/williamghelfi/Work/React/react-router/website/node_modules/babel-core/lib/transformation/pipeline.js:46:16)
    at transpile (/Users/williamghelfi/Work/React/react-router/website/node_modules/babel-loader/lib/index.js:50:20)
    at Object.module.exports (/Users/williamghelfi/Work/React/react-router/website/node_modules/babel-loader/lib/index.js:173:20)
```

Reply:
Yeah, sorry! This PR https://github.com/ReactTraining/react-router/pull/7025 should have a fix for that. I'll verify that it works and, if it does, get that merged in.

Reply:
Okay, I merged that in, so you should be able to verify the fix locally.

Reply:
Ok thanks, let me check if the new syntax I tried actually works on GitHub

Reply:
Hey @pshrmn , I think it works on both envs now 👍